src/Stryker.Core/Stryker.Core.UnitTest/Initialisation/InputFileResolverTests.cs

```diff
@@ -141,12 +143,56 @@ public void InputFileResolver_InitializeShouldIgnoreBinFolder(string folderName)
 
             var target = new InputFileResolver(fileSystem);
 
-            var result = target.ResolveInput(@"c:\TestProject\", "");
+            var result = target.ResolveInput(@"c:\TestProject\", "", new List<string>());
 
             result.TestProjectPath.ShouldBe(@"c:\TestProject\");
             result.ProjectContents.Children.Count.ShouldBe(1);
         }
 
+        [SkippableFact]
```

Comment:
Yes, due to the fact that windows statements are used and all the other tests are also skippablefact :-). I think that they are there for a reason isn't it?

Reply:
We currently have a PR open to resolve the skippable tests, making them work on platforms other than Windows, so it would be nice to not introduce a new windows-only test. What windows only features are used?

Reply:
Ah! #197 you mean? Good point, to be honest I saw these tests and I extended it for my specific testing purposes. I can't actually tell you what kind of specific windows feature is being used. Do you want that I use already the basic setup of the PR 197 or will the PR suppliers change this accordingly?

Reply:
#197 only needs a small fix before it can be merged so I don't expect that to take long. If this PR is merged sooner though then I suppose it would be on them to fix your test. I would prefer if it was fixed in this PR ;) I suggest you work on the other comments first and see how it goes.